assets/src/dashboard/app/views/exploreTemplates/index.js

```diff
@@ -61,6 +64,16 @@ function ExploreTemplates() {
     });
   }, [templatesOrderById, templates]);
 
+  const handlePreviewTemplate = useCallback(
+    (template) => {
+      if (!enableTemplatePreviews) {
+        return () => {};
+      }
+      return preview.set(template);
```

Comment:
Nit: There is a need to return anything from this function as `preview.set(template)` returns `undefined` so this can more simply be: 

```javascript
if (enableTemplatePreviews) {
  preview.set(template);
}
```